legate/timing/timing.py

```diff
@@ -53,17 +55,15 @@ def measure_nanoseconds(self) -> Future:
         )
 
 
-class Time(object):
+class Time(LegateDataInterface):
     def __init__(self, future: Future, dtype: Any) -> None:
         self.future = future
         self.dtype = dtype
         self.value: Union[int, float, None] = None
 
     @property
-    def __legate_data_interface__(self) -> dict[str, Any]:
-        result: dict[str, Any] = {"version": 1, "data": dict()}
-        result["data"]["Legate Timestamp"] = self
-        return result
+    def __legate_data_interface__(self) -> LegateDataInterfaceItem:
+        return {"version": 1, "data": dict()}
```

Comment:
@magnatelee Do you still want this to expose the legate data interface? If so, we will need to wrap its Future in a Store, e.g. something like`"data": {pa.Field(...), self.convert_to_store()}`. I don't think this is worth the trouble.

Reply:
I agree. let's just remove `__legate_data_interface__`.

Reply:
https://github.com/nv-legate/legate.core/pull/494/commits/e880767d5d01fa04a541baba3f223c9f14154241